tests/export/image_resize/test_image_resize.py

```diff
@@ -0,0 +1,175 @@
+        os.path.abspath(os.path.dirname(__file__)),
+        '..',
+        '..',
+        'fixtures',
+        'images',
+        img_name,
+    )
+
+    if as_binary:
+        with open(file_path, 'rb') as f:
+            return f.read()
+
+
+class GetImageFromSrcTestCase(TestCase):
+    def test_get_image_from_src_url(self):
+        web_data = get_image_from_src('http://httpbin.org/image/png')
+        local_data = get_img('image1.png', as_binary=True)
+
+        self.assertEqual(web_data, local_data)
+
+    def test_get_image_from_src_data(self):
+        img_data = get_img('image1.data', as_binary=True)
+        result = get_image_from_src(img_data)
+
+        self.assertEqual(img_data, result)
+
+
+class GetImageDataAndFileNameTestCase(TestCase):
+    def test_get_image_from_src_url(self):
+        img_data, filename = get_image_data_and_filename('http://httpbin.org/image/png', 'png.png')
+
+        self.assertEqual(img_data, img_data)
+        self.assertEqual(filename, 'png')
+
+
+class ImageResizerTestCase(TestCase):
+    def test_init(self):
+        image_data = get_img('image1.png', as_binary=True)
+
+        ir = ImageResizer(image_data, 'image1.png', '100 px', '100 px')
+
+        self.assertTrue(ir)
+
+    def test_init_object(self):
+        image_data = get_img('image1.png', as_binary=True)
+        ir = ImageResizer(image_data, 'image1.png', '100 px', '100 px')
+
+        self.assertTrue(ir)
+
+    def test_has_skipable_extension_tue(self):
+        image_data = get_img('image1.png', as_binary=True)
+        ir = ImageResizer(image_data, 'image1.png', '100 px', '100 px', skip_extensions=['png'])
+
+        self.assertTrue(ir.has_skipable_extension())
+
+    def test_has_skipable_extension_false(self):
+        image_data = get_img('image1.png', as_binary=True)
+        ir = ImageResizer(image_data, 'image1.png', '100 px', '100 px')
+
+        self.assertFalse(ir.has_skipable_extension())
+
+    def test_has_height_and_width_true(self):
+        image_data = get_img('image1.png', as_binary=True)
+        ir = ImageResizer(image_data, 'image1.png', '10 px', '20 px')
+
+        self.assertTrue(ir.has_height_and_width())
+
+    def test_has_height_and_width_false(self):
+        image_data = get_img('image1.png', as_binary=True)
+
+        ir = ImageResizer(image_data, 'image1.png', '0 px', '10 px')
+        self.assertFalse(ir.has_height_and_width())
+
+        ir = ImageResizer(image_data, 'image1.png', '10 px', '0 px')
+        self.assertFalse(ir.has_height_and_width())
+
+        ir = ImageResizer(image_data, 'image1.png', '0 px', '0 px')
+        self.assertFalse(ir.has_height_and_width())
+
+    def test_init_image(self):
+        image_data = get_img('image1.png', as_binary=True)
+        ir = ImageResizer(image_data, 'image1.png', '100 px', '100 px')
+        ir.init_image()
+        self.assertEqual(ir.image, Image.open(StringIO(image_data)))
+
+        image_data = get_img('image1.data', as_binary=True)
```

Comment:
These next 4 lines seem redundant. Can you explain why they are different than lines 98-101?